Create 2024-07-15.md (#164)
department-of-veterans-affairs/developer-portal-content: content/direct-deposit-management/release-notes/2024-06-28.md

> We have added indicators for payment types in the response for the Direct Deposit API. This will support the ability to differentiate between different types of direct deposits.  

Reviewer: ```suggestion
We've added indicators for payment types in the response for the Direct Deposit API. This will support the ability to differentiate between different types of direct deposits.  
```
